exercises/settings.gradle

```diff
@@ -20,6 +20,7 @@ include 'concept:salary-calculator'
 include 'concept:squeaky-clean'
 include 'concept:tim-from-marketing'
 include 'concept:wizards-and-warriors'
+include 'concept:arcade-high-score'
```

Comment:
The list of exercises is now sorted alphabetically to limit the number of merge conflicts occurring when multiple people are working on new concept exercises at the same time.

Could you move this entry to the appropriate place?